deploy-board/deploy_board/webapp/capacity_views.py

```diff
@@ -48,7 +48,9 @@ def get(self, request, name, stage):
             if basic_cluster_info:
                 base_image_id = basic_cluster_info.get('baseImageId')
                 base_image = baseimages_helper.get_by_id(request, base_image_id)
-                asg_cluster = autoscaling_groups_helper.get_group_info(request, cluster_name)
+                
+                # asg_cluster = autoscaling_groups_helper.get_group_info(request, cluster_name) NO COMMIT
```

Comment:
nit: remove commented code

Reply:
yeah, forgot to remove some that I put few months ago.